util/logtools/splitter.ts

```diff
@@ -89,20 +143,23 @@ export default class Splitter {
       lines.push(line);
     for (const line of Object.values(this.addedCombatants))
       lines.push(line);
+    for (const line of Object.values(this.rsvLines))
+      lines.push(line);
     lines.push(line);
 
     lines = lines.sort((a, b) => {
-      // Sort by earliest time first, then by earliest log id.
+      // Sort by earliest time first, then by the lowest-numbered type.
       // This makes the log a little bit fake but maybe it's good enough.
-      const aStr = a.slice(3, 36) + a.slice(0, 3);
-      const bStr = b.slice(3, 36) + b.slice(0, 3);
+      const aStr = (a.split('|')[1] ?? '') + (a.split('|')[0] ?? '');
+      const bStr = (b.split('|')[1] ?? '') + (b.split('|')[0] ?? '');
```

Comment:
What are these changes for?

Reply:
These slices don't work (or at least don't work as intended, I think), given that the logtype field can be either 2 or 3 characters long (eg all of the OverlayPlugin-emitted lines).  So rather than arbitrarily splitting the line at a fixed length, it splits by field (which I think was the original intent?)

Reply:
Oh, oops!